### ISSUE TITLE
Point cloud range - is it MAX_DEPTH?

### ISSUE THREAD

**Issue by Sreeni1204** [zhixinwang/frustum-convnet#31]
Hello

In paper it is claimed that "We consider a depth range of [0, 70] meters in KITTI" How do i change this value? 

Is changing MAX_DEPTH in config file enough?

**Comment by zhixinwang** [comment]
Yes.  Please use latest code, I fixed the setting by this [comment](https://github.com/zhixinwang/frustum-convnet/commit/eb8e67fbbf21f21aba4616d513957c4a8bc6f9fd).

**Comment by Sreeni1204** [comment]
Cool mate, I have made this change locally and have used it.

Thank you for the response :)
